tools/png2obj/tests/samples/4x4_chessboard_1.obj

```diff
@@ -0,0 +1,95 @@
+v 0.000000 0.000000 1.000000
```

Comment:
It's relatively easy to read compressed files via `gzip` module. Could we gzip this one, so it doesn't figure out as plain text?